src/content-switcher/content-switcher.stories.ts

```diff
@@ -17,21 +17,19 @@ storiesOf("Components|Content Switcher", module)
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
 		template: `
-			<a href="https://builder.carbondesignsystem.com/from-json/%7B%22title%22&#13;
-			%3A%22Content%20switcher%22%2C%22data%22%3A%7B%22items%22%3A%5B%7B%22&#13;
-			type%22%3A%22content-switcher%22%2C%22size%22%3A%22sm%22%2C%22&#13;
-			selectedIndex%22%3A0%2C%22items%22%3A%5B%7B%22name%22%3A%22&#13;
-			first%22%2C%22text%22%3A%22First%20section%22%2C%22disabled%22%3Afalse%2C%22&#13;
-			type%22%3A%22switch-item%22%2C%22id%22%3A%223%22%2C%22codeContext%22%3A%7B%22&#13;
-			name%22%3A%22switch-item-3%22%7D%7D%2C%7B%22name%22%3A%22second%22%2C%22text%22&#13;
-			%3A%22Second%20section%22%2C%22disabled%22%3Afalse%2C%22type%22%3A%22&#13;
-			switch-item%22%2C%22id%22%3A%224%22%2C%22codeContext%22%3A%7B%22name%22%3A%22&#13;
-			switch-item-4%22%7D%7D%2C%7B%22name%22%3A%22third%22%2C%22text%22%3A%22Third%20&#13;
-			section%22%2C%22disabled%22%3Afalse%2C%22type%22%3A%22switch-item%22%2C%22id%22&#13;
-			%3A%225%22%2C%22codeContext%22%3A%7B%22name%22%3A%22switch-item-5%22%7D%7D%5D%2C&#13;
-			%22id%22%3A%222%22%2C%22codeContext%22%3A%7B%22name%22%3A%22content-switcher-2%22&#13;
-			%7D%2C%22cssClasses%22%3A%5B%5D%7D%5D%2C%22id%22%3A1%7D%2C%22cssClasses%22&#13;
-			%3A%5B%5D%2C%22allCssClasses%22%3A%5B%5D%7D" target="_blank">
+			<a href="https://builder.carbondesignsystem.com/from-json/%7B%22title%22%3A%22&#13
+			ContentSwitcherFragment%22%2C%22data%22%3A%7B%22items%22%3A%5B%7B%22type%22%3A%22&#13
+			content-switcher%22%2C%22size%22%3A%22sm%22%2C%22selectedIndex%22%3A0%2C%22items&#13
+			%22%3A%5B%7B%22name%22%3A%22first%22%2C%22text%22%3A%22First%20section%22%2C%22&#13
+			disabled%22%3Afalse%2C%22type%22%3A%22switch-item%22%2C%22id%22%3A%223%22%2C%22&#13
+			codeContext%22%3A%7B%22name%22%3A%22switch-item-3%22%7D%7D%2C%7B%22name%22%3A%22&#13
+			second%22%2C%22text%22%3A%22Second%20section%22%2C%22disabled%22%3Afalse%2C%22type&#13
+			%22%3A%22switch-item%22%2C%22id%22%3A%224%22%2C%22codeContext%22%3A%7B%22name%22&#13
+			%3A%22switch-item-4%22%7D%7D%2C%7B%22name%22%3A%22third%22%2C%22text%22%3A%22&#13
+			Third%20section%22%2C%22disabled%22%3Afalse%2C%22type%22%3A%22switch-item%22%2C%22id&#13
+			%22%3A%225%22%2C%22codeContext%22%3A%7B%22name%22%3A%22switch-item-5%22%7D%7D%5D%2C%22id&#13
+			%22%3A%222%22%2C%22codeContext%22%3A%7B%22name%22%3A%22content-switcher-2%22%7D%7D%5D%2C&#13
+			%22id%22%3A1%7D%2C%22allCssClasses%22%3A%5B%5D%7D" target="_blank">
```

Comment:
This is breaking the story.

Reply:
I think it should have semi-colons instead of &#13? Was right in the previous PR